core/src/main/java/org/apache/calcite/sql/type/SqlTypeUtil.java

```diff
@@ -1009,6 +1009,7 @@ private static boolean flattenFields(
     return nested;
   }
 
+
   /**
```

Comment:
No need to add this line ?

Reply:
Sorry,  I made some modification and recover it , but left new line and space.